src/main/java/com/cocodan/triplan/schedule/service/ScheduleService.java

```diff
@@ -38,9 +37,17 @@
     public Long createSchedule(ScheduleCreationRequest scheduleCreationRequest, Long memberId) {
         Schedule schedule = scheduleConverter.convertSchedule(scheduleCreationRequest, memberId);
 
+        scheduleCreationRequest.getDailyScheduleSpotCreationRequests().stream()
+                .filter(this::doesNotSavedSpot)
+                .forEach(spotService::createSpot);
+
         return scheduleRepository.save(schedule).getId();
     }
 
+    private boolean doesNotSavedSpot(DailyScheduleSpotCreationRequest dailyScheduleSpotCreationRequest) {
```

Comment:
```suggestion
    private boolean isAlreadySavedSpot(DailyScheduleSpotCreationRequest dailyScheduleSpotCreationRequest) {
```
으로 하고 ! 떼고 반환하면 어떨까요?

Reply:
이렇게 하면 호출하는 쪽에서 !isAlreadySavedSpot() 으로 호출하게 되서 저는 현재 방법이 취향에 맞네요~~